docs/installation/vault.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Vault
+---
+
+## Installing
+
+To install Vault follow the
+[Vault installation](https://www.vaultproject.io/docs/install) instructions. If
+you have a mac I would suggest installing
+[Vault using Homebrew](https://formulae.brew.sh/formula/vault). You can validate
+that you have it installed by running the following command.
+
+```
+vault -h
+```
+
+After Vault has been installed, the next step is to startup a Vault server.
+
+```
+vault server -dev
+```
+
+This will give you an output that looks like this:
+
+```
+==> Vault server configuration:
+
+             Api Address: http://127.0.0.1:8200
+                     Cgo: disabled
+         Cluster Address: https://127.0.0.1:8201
+              Listener 1: tcp (addr: "127.0.0.1:8200", cluster address: "127.0.0.1:8201", max_request_duration: "1m30s", max_request_size: "33554432", tls: "disabled")
+               Log Level: info
+                   Mlock: supported: false, enabled: false
+           Recovery Mode: false
+                 Storage: inmem
+                 Version: Vault v1.4.2
+             Version Sha: 18f1c494be8b06788c2fdda1a4296eb3c4b174ce+CHANGES
+
+WARNING! dev mode is enabled! In this mode, Vault runs entirely in-memory
+and starts unsealed with a single unseal key. The root token is already
+authenticated to the CLI, so you can immediately begin using Vault.
+
+You may need to set the following environment variable:
+
+    $ export VAULT_ADDR='http://127.0.0.1:8200'
+
+The unseal key and root token are displayed below in case you want to
+seal/unseal the Vault or re-authenticate.
+
+Unseal Key: <a long key>
+Root Token: <a token>
+
+Development mode should NOT be used in production installations!
+```
+
+## Configuring
+
+Once Vault is installed and running, the next step is to configure it to work
+with your application. Since we want multiple applications(in production) to be
+able to use the same Vault infrastructure we control access by using
+[Vault policies](https://www.vaultproject.io/docs/concepts/policies) and
+different
+[Key/Value secret paths](https://www.vaultproject.io/docs/secrets/kv/kv-v2#setup).
```

Comment:
```suggestion
[key/value secret paths](https://www.vaultproject.io/docs/secrets/kv/kv-v2#setup).
```

The linked documentation refers to it as `key/value` so I think we should follow the capitalization set forth by Hashicorp.